src/js/patch.ts

```diff
@@ -134,6 +134,7 @@ function defines(flags: FlagSet,
     _PROGRESSIVE_BRACELET: true,
     _RABBIT_BOOTS_CHARGE_WHILE_WALKING: flags.rabbitBootsChargeWhileWalking(),
     _RANDOM_FLYER_SPAWNS: true,
+    _RANDOM_WILD_WARP: flags.randomizeWildWarp(),
```

Comment:
_RANDOM_WILD_WARP_COUNT?  We already have a setting to just randomize the locations and it doesn't require the change here?

Is the actual list available here?  Might be better to actually just look at the number - something like `_UPDATE_WILD_WARP_COUNT: rom.wildWarps.length !== 16` or something?

Reply:
This function doesn't currently take rom as an argument, but it certainly could.  Right now it only has access to flags and 'early' or 'late'.